app/controllers/edamams_controller.rb

```diff
@@ -0,0 +1,15 @@
+class EdamamsController < ApplicationController
+
+  def index
+    recipe_list = RecipeApiWrapper.recipes(params[:recipe])
+    @recipe_list = Kaminari.paginate_array(recipe_list).page(params[:page]).per(10)
+  end
```

Comment:
You've written your API wrapper to raise an error if the API call fails, but you're not looking for an error here. You should wrap this call in a `begin`/`rescue`.